tests/integration_tests/build/test_coverage.py

```diff
@@ -23,7 +23,7 @@
 # this contains the frequency while on AMD it does not.
 # Checkout the cpuid crate. In the future other
 # differences may appear.
-COVERAGE_DICT = {"Intel": 85.08, "AMD": 84.28}
+COVERAGE_DICT = {"Intel": 85.34, "AMD": 84.58}
```

Comment:
Once all these utils go to their respective `test_utils.rs` files, you can update the `exclude_pattern` in `tests/integration_tests/build/test_coverage.py` to exclude this code from coverage.

Our production code coverage should not include test utils.